exit_db/migrations/2018-02-27-043631_add_tables/up.sql

```diff
@@ -6,5 +6,6 @@ CREATE TABLE clients (
   luci_pass VARCHAR NOT NULL,
   internal_ip VARCHAR NOT NULL,
   email VARCHAR NOT NULL,
-  zip VARCHAR NOT NULL
+  zip VARCHAR NOT NULL,
+  country VARCHAR NOT NULL
```

Comment:
Not all countries have ZIPs, I wonder if we could just convert stuff to coordinates somehow, or a system like what3words, Makaney or Open Location Code

Reply:
Geohash and Open Location Code have Rust implementations

Reply:
I just wanted to call it postal code. As that's pretty relevant. Although some countries don't even have that at which point we can leave it blank. We don't really need postal codes